build/Targets/GenerateInternalsVisibleTo.targets

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="Windows-1252"?>
```

Comment:
Guessing this change is what broke the LInux job.  Don't know why this was added.  Should be removed and make sure you didn't add a new BOM to this file.